buildscripts/README.md

```diff
@@ -0,0 +1,97 @@
+    * PHP 7.0* or PHP 7.1*, install Visual Studio 2015 and make sure C++ tools are enabled. 
+    * PHP 7.2*, install Visual Studio 2017, including Visual C++ toolset, the Windows SDK components, and Git for Windows. 
+
+To use the sample build scripts `builddrivers.py` and `buildtools.py`, install Python 3.x in Windows. 
+
+## Compile the drivers 
+
+You must first be able to build PHP 7.* without including these extensions. For help with doing this, see the [official PHP website](https://wiki.php.net/internals/windows/stepbystepbuild) for building PHP 7.0* or PHP 7.1* on Windows or [PHP SDK page](https://github.com/OSTC/php-sdk-binary-tools) for the new instructions for building PHP 7.2 and/or above.
+
+The Microsoft Drivers for PHP for SQL Server have been compiled and tested with PHP 7.0.* and 7.1.* using the Visual C++ 2015 as well as PHP 7.2.0 beta using the Visual C++ 2017 v15.0. 
+
+### Manually building from source 
+
+1. Download the *source* directory from this repository
+
+2. Make a copy of the *shared* folder as a subfolder in *sqlsrv* and/or *pdo_sqlsrv* folder
+
+3. Copy the *sqlsrv* and/or *pdo_sqlsrv* folder(s) into the PHP source ext subdirectory
+
+4. Run `buildconf --force` to rebuild the configure.js script to include the *sqlsrv* and/or *pdo_sqlsrv* driver(s).
+
+5. Run `configure.bat` with the desired driver options (as shown below) to generate the makefile. You can run `configure.bat --help` to see what other options are available. For example, for non-thread safe build, add this option `--disable-zts`.  
+    * For SQLSRV add: `--enable-sqlsrv=shared`
+    * For PDO_SQLSRV add: `--enable-pdo --with-pdo-sqlsrv=shared`
+
+6. Run `nmake`. Optionally, you can run `nmake clean` first.
+
+7. To install the drivers, there are two ways:
+    * Run `nmake install`, or
+    * Copy the drivers:
+        * Find the directory where the newly compiled `php.exe` is
+        * Locate the compiled php_sqlsrv.dll and/or php_pdo_sqlsrv.dll 
+        * Copy the dll(s) to the `ext` subfolder  
+
+### Using the sample build scripts
+
+The sample build scripts, `builddrivers.py` and `buildtools.py`, are expected to build our extensions for PHP in Windows.
+
+#### Overview
+
+When asked to provide the PHP version, you should enter values like `7.1.7`. If it's alpha, beta, or RC, make sure the name you provide matches the PHP tag name without the prefix `php-`. For example, for PHP 7.2 beta 2, the tag name is `php-7.2.0beta2`, so you should enter `7.2.0beta2`. Visit [PHP SRC]( https://github.com/php/php-src) to find the appropriate tag names.
+
+It's recommended that the PHP SDK is unzipped into the shortest possible path, preferrably somewhere near the root drive. Therefore, this script will create a `php-sdk` folder in the C:\ drive. This `php-sdk` folder will remain unless you remove it yourself. For ongoing development, it's suggested you keep it around. The build scripts will handle updating the PHP SDK if new version is available. 
```

Comment:
'It's' here too... though this could be changed to say who 'it' refers to. PHP developers presumably? Something like 'PHP recommends that the PHP SDK...', 'For ongoing development, PHP suggests..' or 'we suggest...' if 'it' refers to us instead.